packages/e2e/README.md

```diff
@@ -44,14 +44,16 @@ Put the a file named `<package_name>_e2e_test.dart` in the app' `test_driver` di
 ```dart
 import 'dart:async';
 import 'dart:io';
+import 'package:e2e/common.dart' as common;
```

Comment:
It seems odd to me to use `common` as the namespace here because there's nothing e2e-specific about that word. Would it be more idiomatic to have the namespace be e2e? e.g. `import 'package:e2e/e2e.dart' as e2e;`

Reply:
removed this code since I implemented the suggestion in the next comment.